src/DvsaLogger/Processor/ApiResponseExtras.php

```diff
@@ -19,16 +17,14 @@
  */
 class ApiResponseExtras implements ProcessorInterface
 {
-    /** @var null|RequestInterface */
-    protected $request = null;
+    /** @var Request */
```

Comment:
Do we want these as inline typing? Or do we want to keep them as dock block types?

Reply:
That's habit from doing it for variables rather than properties. Fixed